src/main/assembly/fakeroot.xml

```diff
@@ -103,6 +103,12 @@ Installed in a fakeroot directory structure.
             <outputDirectory>/etc/init.d</outputDirectory>
             <fileMode>0755</fileMode>
         </fileSet>
+	<!-- .info files -->
+        <fileSet>
+            <directory>src/main/infofiles</directory>
+            <outputDirectory>/etc/grid-security/certificates</outputDirectory>
```

Comment:
I've discussed with DavidG and we could try to push them via the EGI policy packages (such as ca-policy-egi-core-1.79-1.noarch.rpm), which are specific to EGI, and actually also produced by David with an EGI hat instead of IGTF hat. The point is that the PIP looks in the 'trustDir' (i.e. normally /etc/grid-security/certificates) for all the info files, and hence that's also where these files should end up, either as file or as symlink. Any other directory does not make sense since they are the same sort of file from the PIP perspective. They are (reasonably) normal info files, only not IGTF.